blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jorge Vicente Cantero and Martin Duhem
+title: "Meet bloop, a fast tool to compile and test your project"
+---
+
+Martin Duhem and I are happy to announce `bloop`, a tool that Scala developers
+can use to compile and test their projects faster without being tied to their
+stock build tool.
+
+## What is bloop?
+
+Bloop is a command-line tool for fast edit/compile/test workflows. Its primary goal is to compile
+and test your project as fast as possible, offering a snappy developer experience.
```

Comment:
Perhaps say: "faster than with your stock build tool"

You also might want to point out that it's specialized on this edit/compile/test loop... That might make it all clearer that we're not talking about a new build tool.

Reply:
You also might want to explicitly state in the intro here that bloop is not a new build tool. Perhaps put that concern to rest right away.

Reply:
Done